res/css/views/elements/_SSOButtons.scss

```diff
@@ -64,11 +78,6 @@ limitations under the License.
     border-color: #3c5a99;
     color: #ffffff;
 }
-.mx_SSOButton.mx_SSOButton_brand_google {
```

Comment:
Hmm, no colours needed for Google now?

Reply:
Indeed, the branding guidelines show it as a colourful logo on a white button. The old figma for it was white logo on red button but thats Google+

Reply:
![image](https://user-images.githubusercontent.com/2403652/106278904-6f97b700-6233-11eb-8452-c919f2bac470.png)